### PR TITLE
Add resourceID based exception to scan_with_exception_to_backend test

### PR DESCRIPTION
**Pull request by avrahams**
DO NOT MERGE -pending on kubesacpe release

### CODE REVIEW

**Comment by dwertent** [comment]
@avrahams Is this ready to merge?

**Comment by avrahams** [comment]
@dwertent it is still waiting for review 
merge was pending on new kubescape release

File: tests_scripts/kubescape/scan.py [hunk 1]
```diff
@@ -218,6 +218,12 @@ def start(self):
         resource = 'apache'
         control_id = "C-0016"
         control_name = "Allow privilege escalation"
+        #second resource exception (by resource ID)
```
Comment (dwertent) [inline tests_scripts/kubescape/scan.py:221]: I see this was implemented in the past, but we should not have the "expected data" hardcoded in the test.

**Review by dwertent — APPROVED** [review]

**Comment by dwertent** [comment]
Overall, I will prefer having a different test object and not having a single test for both types of exceptions

**Pull request merged**